src/app/components/Unlock/index.jsx

```diff
@@ -33,20 +35,26 @@ class Unlock extends React.Component {
 
   render() {
     return (
-      <div>
+      <div class="unlock--container">
+        <UnlockTwoTone
+          className="unlock--container__icon"
+          twoToneColor="#1890ff"
```

Comment:
Do you think we can/should define the colors somehow globally in variables?

Reply:
fixed in https://github.com/bumi/lightning-browser-extension/pull/41/commits/d92f9576f087d16e5d86ad96c0894d76582c339a